students/kondel_tomasz/lesson_04_unit_testing/fibonacci_test.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import sys
+sys.path.insert(0, '../lesson_03_funtions/')
+from fibonacci_numbers import fib  # noqa: E402
+
+
+class MyUnitTest(unittest.TestCase):
+
+    def test_assertRaises(self):
+        self.assertRaises(fib('aoeu'))
+
+    def test_output_validation_for_8(self):
```

Comment:
`test_fib_is_correct_for_8`. You are not checking a validation code here. Passing of the test is a statement about the code.